evap/staff/fixtures/excel_files_test_data.py

```diff
@@ -133,6 +133,15 @@
     ]
 }
 
+test_enrollment_data_empty_filedata = {
+    'MA Belegungen': [
+        ['Degree', 'Student last name', 'Student first name', 'Student email address', 'Course kind', 'Course is graded', 'Course name (de)', 'Course name (en)', 'Responsible title', 'Responsible last name', 'Responsible first name', 'Responsible email address'],
+    ],
+    'BA Belegungen': [
+        ['Degree', 'Student last name', 'Student first name', 'Student email address', 'Course kind', 'Course is graded', 'Course name (de)', 'Course name (en)', 'Responsible title', 'Responsible last name', 'Responsible first name', 'Responsible email address'],
+    ]
+}
+
```

Comment:
@niklasmohrin For maintainability, we should maybe consider reducing the count of "excel file data" struct we have here, and instead maybe rely on a single fixed "correct" input data set that we modify to the needs of each individual test (e.g. inserting duplicate rows, this here could be just removing all rows except for the headers).

Reply:
I am fine with whatever is simpler. I was wondering if we should try to not generate test data in the tests and instead have functions here that have a name to suggest what kind of modified data they return, but I am not sure if that would lead to too much test specifics bleeding into this one file all shuffled. I suppose there is only one way to find out 😅

I agree to not try it in this PR